### DISCUSSION TITLE
在DB-GPT中使用Chatglm3-6b

### DISCUSSION THREAD

**Discussion by 769713504** [zai-org/ChatGLM3#1032] [Q&A] [answered]
在dbgpt中使用Chatglm3时为啥有的问题在回答完成之后不会停止, 还会输出一些自问自答的内容.
![240326094434](https://github.com/THUDM/ChatGLM3/assets/55389357/8d3d2d8d-4ff5-47db-836c-895c53a7fd0a)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
确定一下是否用了chattemplate，vllm底座的更是要注意
